lib/chef/resource/chocolatey_package.rb

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
We need to set the resource name and need to declare our `provides` and this provider is written more inline with the rest of the package providers, not like windows_package.